docs/current_docs/faq.mdx

```diff
@@ -137,6 +137,10 @@ No. "Rootless mode" means running the Dagger Engine as a container without the `
 
 Yes. By replacing the `/etc/dagger/engine.toml` file with your custom configuration, you can configure the Dagger Engine with CA certs, registry mirrors or additional privileges. However, note that this is a temporary solution and may change in future. [Read more about Dagger Engine configuration](https://github.com/dagger/dagger/blob/main/core/docs/d7yxc-operator_manual.md).
 
+### Can I use Dagger Engine to build Windows Containers?
+
+Unfortunately, not right now. Dagger runs on top of BuildKit and support for Windows Containers is still experimental in BuildKit. In addition, Dagger has a lot of custom code written on top of BuildKit such as networking, init systems, and Linux namespace entries, that do not have exact parallels on Windows.
```

Comment:
This used to be in our docs but got lost during the recent module renovation I think: we *do* support some limited operations on Windows container images; namely just purely file operations. It's mainly just `withExec` that doesn't work.

This is covered in our integ tests ([here](https://github.com/dagger/dagger/blob/701705f67f11518b89f4ad4ebeb65a553d733f09/core/integration/platform_test.go#L201)) so I feel comfortable mentioning it if you think that level of detail is worth it.